authorize.py

```diff
@@ -70,19 +75,35 @@ def authorize_then_consume(event, context):
             rv['statusCode'] = 400
     return rv
 
-def create_user(event, context, mlh = False):
+@ensure_schema({
+    "type": "object",
+    "properties": {
+        "email": {"type": "string", "format": "email"},
+        "password": {"type": "string"},
+        "link": {"type": "string"},
+        "github": {"type": "string"},
+        "major": {"type": "string"},
+        "short_answer": {"type": "string"},
+        "shirt_size": {"type": "string"},
+        "first_name": {"type": "string"},
+        "last_name": {"type": "string"},
+        "dietary_restrictions": {"type": "string"},
+        "special_needs": {"type": "string"},
+        "date_of_birth": {"type": "string"},
+        "school": {"type": "string"},
+        "grad_year": {"type": "string"},
```

Comment:
this should prob be an in

Reply:
Yep. When that happens, I'll make it automatically restrict to current seniors onwards.

It's a string to not mess with the frontend.

Reply:
I see, ok we can make issues on front to change for next hackru